boxsdk/pagination/page.py

```diff
@@ -38,7 +39,7 @@ def _translator(self):
 
         :rtype:   :class:`Translator`
         """
-        return self._session.translator
+        return Translator()
```

Comment:
I'm not sure what this change is doing / how it's related to the PR? Could you explain in the PR comment?